lvsfunc/util.py

```diff
@@ -174,19 +214,20 @@ def replace_ranges(clip_a: vs.VideoNode,
     if ranges is None:
         return clip_a
 
-    out = clip_a
+    out_indicies = list(zip([0] * clip_a.num_frames, range(clip_a.num_frames)))
+    clip_b_indices = list(zip([1] * clip_b.num_frames, range(clip_b.num_frames)))
```

Comment:
probably better to just generate the indices we need for clip_b inside the loop instead of slicing a precomputed list

Reply:
Inside the loop meaning inside the frameeval? Not sure I understand this point.